Content.IntegrationTests/Tests/StorageTest.cs

```diff
@@ -32,9 +32,11 @@ public async Task StorageSizeArbitrageTest()
                 {
                     if (!proto.TryGetComponent<StorageComponent>("Storage", out var storage) ||
                         storage.Whitelist != null ||
-                        !proto.TryGetComponent<ItemComponent>("Item", out var item)) continue;
+                        storage.MaxItemSize == null ||
+                        !proto.TryGetComponent<ItemComponent>("Item", out var item))
+                        continue;
```

Comment:
Would be easier to just use compfactory to grab the compname but not gonna nitpick the pr over it.